client/a8c-for-agencies/sections/woopayments/dashboard-content/site-columns.tsx

```diff
@@ -62,12 +93,62 @@ export const WooPaymentsStatusColumn = ( { state, siteId }: { state: string; sit
 		return null;
 	}
 
+	const popoverContent = (
+		<div className="woopayments-status-popover">
+			<p className="woopayments-status-popover__text">
+				{ translate(
+					'This WooPayments site is not eligible for commission since it was connected after the incentive expiration date.'
+				) }
+			</p>
+			<a
+				href={ localizeUrl(
+					'https://agencieshelp.automattic.com/knowledge-base/automattic-for-agencies-earnings/'
+				) }
+				className="woopayments-status-popover__link"
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				{ translate( 'Learn more about the incentive' ) }
+				<Gridicon icon="external" size={ 16 } />
+			</a>
```

Comment:
We should try using the WordPress as much as possible. 

```suggestion
			<Button
					variant="link"
					className="woopayments-status-popover__link"
					href={ localizeUrl(
					'https://agencieshelp.automattic.com/knowledge-base/automattic-for-agencies-earnings/'
				    ) }
					target="_blank"
				>
					<>
						{ translate( 'Learn more about the incentive' ) }
						<Icon icon={ external } size={ 18 } />
					</>
				</Button>
```